.github/workflows/release.yml

```diff
@@ -0,0 +1,83 @@
+name: release
+
+on: push
+
+jobs:
+  ubuntu-release:
+    runs-on: ubuntu-latest
+    env:
+      CARGO_HOME: /home/runner/.cargo
+      PREFIX: /home/runner/install
+    steps:
+      - name: Linux Cargo Cache
+        id: linux-cache-cargo
+        uses: actions/cache@v4
+        with:
+          path: /home/runner/.cargo
+          key: linux-cache-cargo
+      - name: Install botan
+        run: sudo apt install -y libbotan-2-dev
```

Comment:
are we ok with installing OEM botan for ubuntu?

Reply:
At the moment, the changes we did to handling RNG/seeding are only relevant to windows, but I assume we have to be ready to fine tune this on any other platform as well. So our own version of Botan all the time.